src/HeaderCell.ts

```diff
@@ -24,9 +24,13 @@ class HeaderCell extends HeaderCellBase<HeaderCellProperties> {
 			onSortRequest
 		} = this.properties;
 
+		// sortDetail: undefined => 'asc'
```

Comment:
If it is inline documentation, it should be noted as such, so it doesn't lead people to believe it is just commented out code.